docs/01-Using-Fleet/04-Adding-hosts.md

```diff
@@ -25,9 +26,22 @@ fleetctl package --type pkg --fleet-url=[YOUR FLEET URL] --enroll-secret=[YOUR E
 
 When you install the generated osquery installer on a host, this host will be automatically enrolled in the specified Fleet instance.
 
+### Signing installers
+
+  >**Note:** Windows signing is not yet supported within `fleetctl` (working on that this week), but the resulting MSI could be signed after building.
```

Comment:
```suggestion
  >**Note:** Currently, the `fleetctl package` command does not provide support for signing and notarizing Windows osquery installers. Windows installers can be signed after building.
```

Reply:
@DominusKelvin can you please move the "Note" section to the bottom of the "Signing installers" section. This is because the information in the "Note" section is usually the least valuable information for the user. 

If you have the time, can you update the "Documentation" section in the `handbook/product.md` file to information contributors to place the "Note" subsection at the end of its respective section. The greyed out "Note" subsection is near the bottom of other sections in the docs. 

I also suggest making the Windows sentence consistent with the following macOS sentence:

> The `fleetctl package` command provides support for signing and notarizing macOS osquery installers via the
`--sign-identity` and `--notarize` flags.

Reply:
> Suggested change 
>   >**Note:** Windows signing is not yet supported within `fleetctl` (working on that this week), but the resulting MSI could be signed after building.
>   >**Note:** Currently, the `fleetctl package` command does not provide support for signing and notarizing Windows osquery installers. Windows installers can be signed after building.

Hey Noah. I might be wrong but I don't think the term "Notarizing" applies to Windows app. I think it's more of a Mac thing.

Reply:
> I might be wrong but I don't think the term "Notarizing" applies to Windows app. I think it's more of a Mac thing.

@DominusKelvin I think you're right. I don't know this. Good catch!

Reply:
So I left out "notarising" and applied the rest of the changes.